test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
remove fixtures and test files. We prefer Rspec with factories. Both of which will be added later. For now, skip committing test files